src/main/java/org/example/repository/UserRepository.java

```diff
@@ -0,0 +1,20 @@
+package org.example.repository;
+
+import jakarta.transaction.Transactional;
+import org.example.model.UserStatus;
+import org.example.model.user.User;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface UserRepository extends CrudRepository<User, UUID> {
+    @Modifying
+    @Transactional
```

Comment:
move this @Transactional to service layer

Reply:
Moved @Transactional to service layer